packages/sdk/scripts/testAbiTypes.ts

```diff
@@ -0,0 +1,25 @@
+import {ethers} from 'ethers'
+
+import {HolographABI} from '../src/constants/abi/develop'
+import {getContract} from '../src/utils/contracts'
+
+async function main() {
+  const rpc = 'https://goerli.infura.io/v3/'
+
+  const rpcProvider = new ethers.JsonRpcProvider(rpc)
+
+  const holograph = getContract<typeof HolographABI>(
```

Comment:
Is `typeof` here just used so `getContract` can return a variable type?

Reply:
We need to pass a generic to this function, and generics are pretty much types. Since `HolographABI` is a variable, we pass its type as `typeof HolographABI`